### PR TITLE
Make contact page functional

### PR DESCRIPTION
**Pull request by VanTaro**
Anyone can now contact Level Hero via email by completing the contact us form. Issue #27.

### CODE REVIEW

File: router/user.js [hunk 1]
```diff
@@ -76,21 +76,26 @@ module.exports = (app) => {
...
+    app.post('/contact', (req, res) => {
+        
         var api_key = process.env.MAILGUN_SECRET;
         var domain = process.env.DOMAIN;
         var mailgun = require('mailgun-js')({apiKey: api_key, domain: domain});
 
         var data = {
             from: req.body.username + 'postmaster@sandbox44bda62170b040ca8df8f0d3c1902ead.mailgun.org',
-            to: 'jordanmavrogeorge@gmail.com',
+            to: req.body.email,
```
Comment (Jordomav) [inline router/user.js]: We dont want the email to be sent to the user, we need to have a contact email that this gets sent to

File: app/contactUs/module.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ angular.module('contactUs')
             });
     });
 
-function contactUsController() {
+function contactUsController($http) {
```
Comment (Jordomav) [inline app/contactUs/module.js]: Dont forget, `API` service :)

**Comment by VanTaro** [comment]
Changes made so that the contents of the contact form is sent to Van instead of the sending user.

**Comment by Jordomav** [comment]
@VanTaro No worries, I know why that is, I believe that my PR should fix that as well

**Pull request merged**